scripting/pugsetup.sp

```diff
@@ -433,6 +439,9 @@ public void OnPluginStart() {
   g_LiveTimerRunning = false;
   ReadSetupOptions();
 
+  g_MapVotePool = new ArrayList(64);
```

Comment:
PLATFORM_MAX_PATH, to be consistent with other map string sizes used elsewhere. (I know there's places with a fixed 64 elsewhere, but those are wrong :) )